docs/Tutorials/Basics/simple-drops.md

```diff
@@ -0,0 +1,280 @@
+
+1) Establishing a connection to the NEAR blockchain.  
+2) Create a set of access keys where the assets will be stored.  
+3) Use the Keypom SDK to create the drop  
+4) Create linkdrops  
+
+The following skeleton code can be used as a starting point:
+``` js
+// Each of the two methods to create this drop will have their own unique set of imports
+
+// Imports used in the Keypom SDK method:
+const { initKeypom, createDrop } = require("keypom-js");
+const { KeyPair, keyStores, connect } = require("near-api-js");
+const path = require("path");
+const homedir = require("os").homedir();
+
+// Imports used in the NEAR-API-JS method:
+const { parseNearAmount, formatNearAmount } = require("near-api-js/lib/utils/format");
+const { KeyPair, keyStores, connect } = require("near-api-js");
+const path = require("path");
+const homedir = require("os").homedir();
+
+
+async function simpleDropKeypom(){
+// STEP 1: Initiate a NEAR connection.
+
+// STEP 2: Create a set of access keys 
+
+// STEP 3: Create the drop.
+
+// STEP 4: Create linkdrops
+}
+
+simpleDropKeypom()
+
+```
+
+---
+
+## Getting Started - Connecting to NEAR
+The first step of creating an Keypom drop is to connect to the NEAR blockchain.
+
+This is done with `NEAR-API-JS` and consists of the following steps:
+
+1) Create an Keystore, which stores your access keys used to sign transactions   
+  * select a network, either `testnet` or `mainnet`  
+  * choose a location for the Keystore, either a folder on your local machine, or an in-memory keystore   
+
+2) Define a NEAR configuration using the Keystore  
+3) Use the configuration to initialize a connection to NEAR  
+
+More information about this process can be found [here](https://docs.near.org/tools/near-api-js/quick-reference#key-store).
+
+:::note
+For simplicity, this tutorial will choose a file-based keystore and point to the `~/.near-credentials` folder on your local machine since this is where most of your keys are stored. For more information about KeyStores, visit NEAR's [official docs](https://docs.near.org/tools/near-api-js/quick-reference#key-store).
+:::
+
+
+```js reference
+https://github.com/keypom/keypom-js/blob/bbe4716ff64dd7a73a6d727a5aea518e8141f60f/docs-examples/keypom-js-sdk/simple-example.js#L8-L25
+```
+
+---
+
+## Creating the Simple Drop
+Now that the connection to the NEAR blockchain is set up, the next step is to create the Keypairs and the drop using the Keypom SDK.
+
+
+In the SDK, this process starts with calling the `initKeypom` function. This function will always be the first function you need to call to interact with the SDK. 
+
+`initKeypom` initializes the SDK to allow for interactions with the Keypom Protocol. Without it, none of the other SDK functions would work as expected. If a NEAR connection is not already present,it will initialize a new one for you. More info on the `initKeypom` function can be found [here](../../keypom-sdk/modules#initkeypom).
```

Comment:
```suggestion
`initKeypom` initializes the SDK to allow for interactions with the Keypom smart contracts. Without it, none of the other SDK functions would work as expected. If a NEAR connection is not already present, it will initialize a new one for you. More info on the `initKeypom` function can be found [here](../../keypom-sdk/modules#initkeypom).
```